### PR TITLE
chore: Update issue templates

### PR DESCRIPTION
**Pull request by LekoArts**
Updates to our issue templates.

- Updated website linkes
- Remove translation (not handled currently at all)
- Advertise Discussions more

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
   To make it easier for us to help you, please include as much useful information as possible.
 
   Useful Links:
-  - Documentation: https://www.gatsbyjs.org/docs/
-  - Contributing: https://www.gatsbyjs.org/contributing/
-
-  Gatsby has several community support channels, try asking your question on:
-
-  - Discord: https://gatsby.dev/discord
-  - Twitter: https://twitter.com/gatsbyjs
```
Comment (vladar) [inline .github/ISSUE_TEMPLATE/documentation.md:17]: Do you remove this intentionally? So, do we always suggest using discussions instead?

Comment (LekoArts) [reply .github/ISSUE_TEMPLATE/documentation.md:17]: It was outdated and also not in the bug_report template anymore, so this just aligns things and moves to singular place (the initial "config.yml" view)

**Review by vladar — APPROVED** [review]
Thanks! 👍

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 contact_links:
   - name: Gatsby Documentation
-    url: https://www.gatsbyjs.org/docs/
+    url: https://www.gatsbyjs.com/docs/
     about: Check out the Gatsby documentation for answers to common questions
   - name: Gatsby Discord
     url: https://gatsby.dev/discord
     about: Ask questions, get help and discuss new things you're building with the Gatsby community
   - name: Gatsby Discussions
     url: https://github.com/gatsbyjs/gatsby/discussions
```
Comment (ascorbic) [inline .github/ISSUE_TEMPLATE/config.yml:10]: How about moving this up to the top, and calling it something like "Question, Help or Feature Request 🤔" to match the names of the issue templates

Comment (LekoArts) [reply .github/ISSUE_TEMPLATE/config.yml:10]: The order is like this due to:

1. Self-Serve
2. More interactive chat
3. Slower chat
4. Our own Support

So ideally people solve their issues by going to the docs

Comment (ascorbic) [reply .github/ISSUE_TEMPLATE/config.yml:10]: I'd agree if it weren't for the fact that the top item is always to open an issue. Encouraging some of those to use discussions instead would be a good thing.

**Pull request merged**